quake1_example/class/func_door.gd

```diff
@@ -4,6 +4,9 @@ class_name QmapbspQuakeFunctionDoor
 # this is written from scratch. No QuakeC code was applied here
 # maybe not accurate to the original approach
 
+const GOLD_KEY_MESSAGE = 'You require the Gold Key!'
```

Comment:
Different themes (tech, wizard, etc.) should have different descriptions, for example tech world uses red and blue key cards instead of gold and silver keys... should probably add those and add code that distinguishes between worlds at some point...